### PR TITLE
Project proposal applying to move Aries Askar to OWF

### PR DESCRIPTION
**Pull request by swcurran**
A project proposal applying to move a component of Hyperledger Aries to OWF.

Askar is an open-source, secure storage and key management service implemented in Rust (with wrappers for Python and JavaScript/TypeScript), designed to support the needs of decentralized trust systems within and beyond the Aries ecosystem. It is a critical component that securely manages cryptographic keys, credentials, and other sensitive data used in decentralized identity frameworks. Askar is the secure storage/key management solution for Credo-TS, ACA-Py, Identus, Aries VCX, and other decentralized trust projects.

@andrewwhitehead, @TelegramSam, @tkuhrt

This PR is associated with #42.

### CODE REVIEW

File: projects/askar.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+3. Open Source Development:  Askar is an open-source project, inviting collaboration and contributions from developers around the world. This approach ensures that the project benefits from diverse input and continuous improvement, fostering innovation and transparency in the development of secure storage solutions.
+4. Support for Decentralized Identity: As a secure storage solution tailored to decentralized identity systems, Askar plays a critical role in enabling decentralized, user-controlled identity management. It ensures that the cryptographic materials required for decentralized identities are stored securely and can be accessed reliably by agents and wallets.
+
+## Code of Conduct
+
+Askar follows the[ Hyperledger Code of Conduct](https://github.com/hyperledger/aries-askar/blob/main/CODE_OF_CONDUCT.md), which is what the OWF code of conduct is based on.
```
Comment (tkuhrt) [inline projects/askar.md:32]:
```suggestion
Askar follows the [Hyperledger Code of Conduct](https://github.com/hyperledger/aries-askar/blob/main/CODE_OF_CONDUCT.md), which is what the OWF code of conduct is based on.  
```

File: projects/askar.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+| Name              | Github           | Organization                             |
+| ----------------- | ---------------- | ---------------------------------------- |
+| Andrew Whitehead  | andrewwhitehead  | Funded by Government of British Columbia |
+| Wade Barnes       | WadeBarnes       | Funded by Government of British Columbia |
+| Berend Sliedrecht | berendsliedrecht | Animo Solutions                          |
+| Timo Glastra      | TimoGlastra      | Animo Solutions                          |
+
+## Proposed Project Governance
+
+The current governance model under Hyperledger is consensus-based. This means that decisions are made through discussions, with the aim of community consensus, as outlined in the[ Aries Project Charter](https://docs.google.com/document/d/1F6RbR7xDaBt5CDJhqLJzR4c1pDJtyPGshp9fy6eVtSM/edit?usp=sharing). In cases where no clear consensus is established, a project Technical Steering Committee, or the maintainers (those with escalated GitHub privileges) are granted a louder voice. This approach has proven effective.
```
Comment (tkuhrt) [inline projects/askar.md:72]:
```suggestion
The current governance model under Hyperledger is consensus-based. This means that decisions are made through discussions, with the aim of community consensus, as outlined in the [Aries Project Charter](https://docs.google.com/document/d/1F6RbR7xDaBt5CDJhqLJzR4c1pDJtyPGshp9fy6eVtSM/edit?usp=sharing). In cases where no clear consensus is established, a project Technical Steering Committee, or the maintainers (those with escalated GitHub privileges) are granted a louder voice. This approach has proven effective.
```

Comment (tkuhrt) [inline projects/askar.md:72]: Is the intention for this project to be distinct from the other Aries proposals and their governance processes when it moves to OWF?

**Review by tkuhrt — CHANGES_REQUESTED** [review]
Thanks for the proposal, @swcurran. I have added this proposal to Wednesday's TAC meeting (September 18th). We will cover this proposal after the ACA-Py proposal.

File: projects/askar.md (hunk 2 above)
Comment (swcurran) [reply projects/askar.md:72]: Askar is proposed to be a separate project at OWF, independent of the other artifacts stemming from the existing Hyperledger Aries project, with separate governance. That reflects the distinct purpose of the project, and community around it.

File: projects/askar.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
+# Askar
+
+## Project Name: Askar
+
+We plan to keep the name Askar, dropping the “Aries” prefix. The project is well known in the global identity community.
+
+## Preferred Maturity Level
+
+The project should at least be a “Growth”, and arguably, “Impact”, given its maturity, robustness, the number of projects (all Aries frameworks, Credo-TS, and others outside of Aries) in which it is embedded, and the myriad of production deployments of those projects. The project is lacking sufficient documentation and the documentation website, which might hold it back from being an “Impact” project.
```
[the remark this answers is not in the document]
Comment (swcurran) [reply projects/askar.md:9]: Added how the Growth/Impact stage requirements are met to the proposal.

**Review by tkuhrt — APPROVED** [review]
@swcurran : Please let us know when this is ready for merge.

**Comment by swcurran** [comment]
Updated to reflect the changed maturity level of "Growth".

**Pull request merged**